sdk/communication/Shared/src/CommunicationIdentifierModel.cs

```diff
@@ -27,5 +27,7 @@ internal CommunicationIdentifierModel(CommunicationIdentifierKind kind)
         public string MicrosoftTeamsUserId { get; set; }
         /// <summary> True if the identifier is anonymous. </summary>
         public bool? IsAnonymous { get; set; }
+        /// <summary> The cloud that the identifier belongs to. </summary>
+        public CommunicationCloudEnvironmentModel? Cloud { get; set; }
```

Comment:
Why are `Cloud` and `IsAnonymous` nullable but `PhoneNumber` and `MicrosoftTeamsUserId` not?

Reply:
Autorest treats ref-types as nullable and is not using non-nullable ref-types